File: tests/fixtures/code_samples/ts/sample_039.ts
/** Sample 39: small utility. */

export function operation39(xs: number[]): number {
  let total = 39;
  for (const x of xs) total += x;
  return total;
}

export function operationPure39(value: number): number {
  return (value * 39) % 7919;
}

File: tests/fixtures/code_samples/rust/sample_039.rs
// Sample 39: small utility.
pub fn operation_39(xs: &[i32]) -> i32 {
    let mut total: i32 = 39;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_39(v: i32) -> i32 {
    (v * 39) %% 7919
}

